phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/apps/AppsController.java

```diff
@@ -85,6 +97,38 @@ public void loadApps() {
         });
 
         this.view.setOnSelectCategory(categoryDTO -> this.view.populateApps(categoryDTO.getApplications(), settingsManager));
+
+        this.view.setOnSetDefaultCategoryIcons(categoryDTOS -> {
+            // set default category icons
+            try {
+                StringBuilder cssBuilder = new StringBuilder();
+                for (CategoryDTO category : categoryDTOS) {
+                    cssBuilder.append("#" + category.getName().toLowerCase() + "Button{\n");
+                    String categoryIcon = category.getIcon();
+                    if (StringUtils.isEmpty(categoryIcon) || ! new File(new URI(categoryIcon)).exists()) {
+                        cssBuilder.append("-fx-background-image: url('/org/phoenicis/javafx/views/common/phoenicis.png');\n");
+                    } else {
+                        cssBuilder.append("-fx-background-image: url('" + categoryIcon + "');\n");
+                    }
+                    cssBuilder.append("}\n");
+                }
+                String css = cssBuilder.toString();
+                Path temp = Files.createTempFile("defaultCategoryIcons", ".css").toAbsolutePath();
+                Files.write(temp, css.getBytes());
+                String defaultCategoryIconsCss = temp.toUri().toString();
+                themeManager.setDefaultCategoryIconsCss(defaultCategoryIconsCss);
+
+                // apply current theme again to fix hierarchy
+                final String shortName = themeManager.getCurrentTheme().getShortName();
+                final String url = String.format("/org/phoenicis/javafx/themes/%s/main.css", shortName);
+                final URL style = this.getClass().getResource(url);
+                this.view.getTabPane().getScene().getStylesheets().clear();
+                this.view.getTabPane().getScene().getStylesheets().addAll(defaultCategoryIconsCss, style.toExternalForm());
```

Comment:
I don't like our current way of interacting with the stylesheets.
Ideally I would like to integrate the handling/interacting in the `ThemeManager` or a new class called `StyleManager` or `CSSManager`.
This class should encapsulate the interaction with stylesheets via an `set()` method, which takes two arguments:

- The first argument is an enum, which tells the method what type of stylesheet you want to set. This value can `DEFAULT_THEME`, `USED_THEME`, `CATEGORY_ICON_THEME` and maybe some other values we'll need.
Depending on the given enum value the method then decides which value inside the stylesheet-list it needs to change.
- The second argument is a File object. This file object is the new stylesheet to be set.

What do you think?